external-crates/move/crates/move-compiler/src/linters/mod.rs

```diff
@@ -75,6 +149,7 @@ pub fn linter_visitors(level: LintLevel) -> Vec<Visitor> {
                 constant_naming::ConstantNamingVisitor.visitor(),
                 unnecessary_while_loop::WhileTrueToLoop.visitor(),
                 meaningless_math_operation::MeaninglessMathOperation.visitor(),
+                unneeded_return::UnneededReturnVisitor.visitor(),
```

Comment:
I kinda want to put this one on by default. But sure, we can leave it here for now

Reply:
On by default is good, but there is a long tail of code in our repo that will start complaining if we enable it.